requirements/uitests.txt

```diff
@@ -71,8 +71,9 @@ zope.interface==4.5.0 \
     --hash=sha256:e6b77ae84f2b8502d99a7855fa33334a1eb6159de45626905cb3e454c023f339 \
     --hash=sha256:a1daf9c5120f3cc6f2b5fef8e1d2a3fb7bbbb20ed4bfdc25bc8364bc62dcf54b \
     --hash=sha256:57c38470d9f57e37afb460c399eb254e7193ac7fb8042bd09bdc001981a9c74c
-pluggy==0.6.0 \
-    --hash=sha256:7f8ae7f5bdf75671a718d2daf0a64b7885f74510bcd98b1a0bb420eb9a9d0cff
+    pluggy==0.7.1 \
+    --hash=sha256:6e3836e39f4d36ae72840833db137f7b7d35105079aee6ec4a62d9f80d594dd1 \
+    --hash=sha256:95eb8364a4708392bae89035f45341871286a333f749c3141c20573d2b3876e1
```

Comment:
indentation :scream:

Reply:
Let's back out this change maybe with a rebase to master?

Reply:
Master doesn't have this updated. I'll get a mismatch error.